### ISSUE TITLE
Unable to resolve module '...' from 'storyLoader.js'

### ISSUE THREAD

**Issue by xareelee** [elderfo/react-native-storybook-loader#23]
I think it is not [react-native-storybook-loader](https://github.com/elderfo/react-native-storybook-loader)'s problem, but I think you may provide some information about this issue.

I put the `.stories.js` file under a `__tests__` folder and refreshed the `storyLoader.js` generated by react-native-storybook-loader, then it shows an error:

<img src="https://cloud.githubusercontent.com/assets/804308/24401875/c31cac2a-13e8-11e7-9a79-a2df1c7e0525.png" width=275 />

I put the same file in a non-`__tests__` folder and refreshed the `storyLoader.js` again, then it goes well.

I did not find any clue about this issue. Do you know why?

**Comment by elderfo** [comment]
Sorry I haven't been able to look into this yet. I should be able to check out this and the other issue you logged this weekend.

**Comment by elderfo** [comment]
It appears `__tests__` folders are excluded by the React Native packager. It looks to be [hard coded](https://github.com/facebook/react-native/blob/7098c450d8274e441135405adf4e83871c74a8af/packager/src/Resolver/index.js#L69-L72).

I also verified by creating a new React Native project and creating a component in a `__tests__` folder and got the same error. 

I suggest logging an issue/feature request on the React Native project (feel free to reference this issue so we can track progress). I do think this is a legitimate use case.

**Comment by elderfo** [comment]
@xareelee, did you open an issue with the React Native team? If so, can you link it here?

**Comment by xareelee** [comment]
Nope, I did not. I'm not sure how to describe this issue in the React Native repo. 

How about that you report this issue there?

**Comment by elderfo** [comment]
Ok, I will log one and link it.

**Comment by BatDroid** [comment]
@elderfo still have the issue. how did it go with the report?
